src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1380,11 +1379,14 @@ From there, simply click on a Markdown file or SVG icon in the File Explorer and
   <data name="Delete_Dialog_Description" xml:space="preserve">
     <value>Are you sure you want to delete this item?</value>
   </data>
-	<data name="Yes" xml:space="preserve">
+  <data name="Yes" xml:space="preserve">
```

Comment:
Do Yes/No dialogs (or buttons) really need this? I mean, I thought it awas possible to use a `MessageBox` function with flags to determine the buttons, and they would be automatically created, localised. Or is that not possible with the languages we use?

Reply:
@Jay-o-Way
Don't forget that we don't support all languages. So using the auto translation of msgbox may lead to inconsistent ux.

Reply:
We are using a ContentDialog and those do not have a default button labels. MessageBox (for WPF) does have that I believe :)